e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -15,17 +15,9 @@ import {
 const embeddingPage = "/admin/settings/embedding-in-other-applications";
 const standalonePath =
   "/admin/settings/embedding-in-other-applications/standalone";
-const licenseUrl = "https://metabase.com/license/embedding";
-const upgradeUrl = "https://www.metabase.com/upgrade";
-const learnEmbeddingUrl =
-  "https://www.metabase.com/learn/embedding/embedding-charts-and-dashboards.html";
-
+const pricingUrl = "https://www.metabase.com/pricing";
```

Comment:
This will actually change again to the upgrade url, it's in milestone 2 and we're waiting for the correct utm tags to use